wzdx/tools/path_history_compression.py

```diff
@@ -0,0 +1,124 @@
+import math
```

Comment:
We only have a single test to cover this file. Should we add a few more to handle both the `generage_compressed_path` as well as the `getChordLength` methods?

Reply:
Added a test for getChordLength and additional tests for generate_compressed_path